Added in location of intent policy and best practice for routing (#73606)
Azure/azure-content: articles/azure-sql/managed-instance/connectivity-architecture-overview.md

- **Sufficient IP addresses:** The SQL Managed Instance subnet must have at least 32 IP addresses. For more information, see [Determine the size of the subnet for SQL Managed Instance](vnet-subnet-determine-size.md). You can deploy managed instances in [the existing network](vnet-existing-add-subnet.md) after you configure it to satisfy [the networking requirements for SQL Managed Instance](#network-requirements). Otherwise, create a [new network and subnet](virtual-network-subnet-create-arm-template.md).

> [!IMPORTANT]

> > When you create a managed instance, a network intent policy is applied on the subnet to prevent noncompliant changes to networking setup, this policy is a hidden resource located in the Virtual Network Resrouce Group. After the last instance is removed from the subnet, the network intent policy is also removed. Rules below are for the informational purposes only, and you should not deploy them using ARM template / PowerShell / CLI. If you want to use the latest official template you could always [retrieve it from the portal](../../azure-resource-manager/templates/quickstart-create-templates-use-the-portal.md). When deploying Auto Failover groups please ensure replication traffic goes direct between the two MI instances and not through a hub network 

Reviewer: please change the wording in a way that implies recommendation, not requirement

> > When you create a managed instance, a network intent policy is applied on the subnet to prevent noncompliant changes to networking setup, this policy is a hidden resource located in the Virtual Network Resrouce Group. After the last instance is removed from the subnet, the network intent policy is also removed. Rules below are for the informational purposes only, and you should not deploy them using ARM template / PowerShell / CLI. If you want to use the latest official template you could always [retrieve it from the portal](../../azure-resource-manager/templates/quickstart-create-templates-use-the-portal.md). When deploying Auto Failover groups please ensure replication traffic goes direct between the two MI instances and not through a hub network 

Reviewer 2: ```suggestion
> When you create a managed instance, a network intent policy is applied on the subnet to prevent noncompliant changes to networking setup. This policy is a hidden resource located in the virtual network of the resource group.  After the last instance is removed from the subnet, the network intent policy is also removed. Rules below are for the informational purposes only, and you should not deploy them using ARM template / PowerShell / CLI. If you want to use the latest official template you could always [retrieve it from the portal](../../azure-resource-manager/templates/quickstart-create-templates-use-the-portal.md). Replication traffic for auto-failover groups between two SQL Managed Instances should be direct, and not through a hub network. 
```

Reviewer 2: @srdan-bozovic-msft does this wording work?

Reviewer: This is awesome @MashaMSFT ! Thx for closing on this long open item :)
